Formula/infer.rb

```diff
@@ -29,14 +29,13 @@ def install
     ENV["OPAMROOT"] = opamroot
     ENV["OPAMYES"] = "1"
 
-    system "opam", "init", "--no-setup"
-    system "opam", "update"
+    # Some of the libraries installed by ./build-infer.sh do not
+    # support parallel builds. Homebrew sets this variable to -j<ncpu>
+    # by default, which would break the build. Moreover,
+    # ./build-infer.sh has its own parallelization logic.
+    ENV["MAKEFLAGS"] = ""
```

Comment:
Does `ENV.j1` work instead? Also, please file issues upstream if there's parallelisation issues as that's a bug that'll massively slow down builds.